app/Http/Controllers/ExportController.php

```diff
@@ -32,7 +42,16 @@ public function __construct(ExportService $export)
      */
     public function show($campaignId)
     {
-        // Compile the data and trigger the CSV download
-        return $this->export->exportSignups($campaignId);
+        // Get the full campaign object.
+        $campaign = $this->campaigns->find($campaignId);
+
+        // Get the email of the authenticated user that made the request.
+        $user = $this->registrar->find(auth()->user()->northstar_id);
```

Comment:
Does `auth()->id()` work here? (I think the answer might be "no", but curious to double-check.)

Reply:
i did work. Grabbing the authorized user in the controller seemed to be the only place I could get it.

Reply:
Oh nice!! I was thinking it might return the local numerical ID. Let's switch it! 🆔

Reply:
Oh wait! I am sorry, i miss understood the question. I will check if `auth()->id()` works for real. I just was confirming that what I had does work 🤦‍♂️

Reply:
Ah ok double checked, and yes, `auth()->id()` returns the local primary key of the user record .

Reply:
Ok cool, I'll make a note for myself to look into making that more useful here.